src/server/scripts/Northrend/Ulduar/Ulduar/ulduar.h

```diff
@@ -368,7 +399,7 @@ enum UlduarData
     DATA_HODIR_YS,
     DATA_THORIM_YS,
     DATA_MIMIRON_YS,
-    DATA_ILLUSION,
+    DATA_ILLUSION, // = 46 used by conditions
```

Comment:
How about explicitly setting it to a high value and adjusting conditions accordingly? Seems more adaptable.